doc/user/content/ingest-data/appendix/mysql-debezium.md

```diff
@@ -6,22 +6,26 @@ aliases:
   - /integrations/cdc-mysql/
   - /connect-sources/cdc-mysql/
   - /ingest-data/cdc-mysql/
+  - /ingest-data/mysql/debezium/
 menu:
   main:
-    parent: "mysql"
-    name: "Using Kafka and Debezium"
+    parent: "ingest-data-appendix"
+    name: "MySQL: Using Kafka and Debezium"
     identifier: "mysql-dbz"
 ---
 
+{{< tip >}}
+- As an alternative to using Kafka and Debezium, Materialize also provides
+  native support for MySQL.
```

Comment:
Once we sort out our docs frontmatter so that there is a landing page under Ingest data/MySQL/, we can update this to link there.